client/src/app/search-result-list/search-result-list.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component} from '@angular/core';
+import {(SearchResultDetailComponent)} from '../search-result-details/search-result-details.component'
```

Comment:
Why the parenthesis in the `SearchResultDetailComponent`?
Also, don't forget the semicolon at the end of the line.
And, add a space between the curly brackets and the `SearchResultDetailComponent`.